runtime/compiler/control/CompilationThread.cpp

```diff
@@ -10891,16 +10891,16 @@ TR::CompilationInfo::debugPrint(char * debugString)
    {
    if (!_traceCompiling)
       return;
-   fprintf(stderr, debugString);
+   fprintf(stderr, "%s", debugString);
    }
 
 void
 TR::CompilationInfo::debugPrint(J9VMThread * vmThread, char * debugString)
    {
    if (!_traceCompiling)
       return;
-   fprintf(stderr, "%8x:", vmThread);
-   fprintf(stderr, debugString);
+   fprintf(stderr, "%8" OMR_PRIxPTR ":", (uintptr_t)vmThread);
```

Comment:
Would it make more sense to use %p since `vmThread` is a pointer?

Reply:
This was to keep the exact same formatting as the original (I don't know if we need to). `%p` would be different, and `%8p` is non-standard if I remember correctly.

Reply:
ok